pyproject.toml

```diff
@@ -67,7 +67,7 @@ dev = [
     "types-PyYAML",
 ]
 docs = [
-    "Sphinx",
+    "Sphinx==8.1.3",
```

Comment:
Perhaps another commit?

Reply:
Yes, rebased on main now to remove it from here 👍